composer.json

```diff
@@ -116,6 +118,8 @@
         "drupal/migrate_upgrade": "^3.0",
         "drupal/restui": "^1.17",
         "drupal/stage_file_proxy": "^1.0@beta",
+        "mglaman/drupal-check": "^1.0",
```

Comment:
Drupal check is installed as part of the lando build process. I originally had it as a composer dependency but had to remove it and pull from the repo as the composer version was broken. Have you verified that the composer version works?

Reply:
It seems to. Or at least, it's struggling with the exact same issues around autoloading some of the classes (think I fixed late last week).

It also works with the global installer mind you... if memory serves I believe I hit problems with autoloading test classes and wondered whether this was due to having a global install of the tool. Moving it into composer felt like a natural first step given that drush and other tools are used in this way to avoid any global config issues.

What I can do today is grab the PRs from upstream, get them to apply and see if this works (or doesn't). While it's nice to have Lando + CI use the same kind of approach, it's probably not that big a deal in practice as Lando fetches the latest release and composer would do something similar... and there's no major issue with global tools on the CI container as its spun up/torn down pretty fast with no cross-over with other apps.

Reply:
Sounds like a plan, I'd check out drupal-check via git and composer and see how current the composer version is as that's what caused me a load of grief a few months back. It's still  in its infancy so a lot of updates and patches are absent from the composer release versions.